packages/integration-test-app/NuGet.Config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same as above. Move wile into the windows folder alongside the solution file.

Reply:
done. Thanks!